src/odemis/acq/fastem.py

```diff
@@ -664,7 +664,7 @@ def acquireTiledArea(stream, stage, area, live_stream=None):
     return f
 
 
-def estimateTiledAcquisitionTime(stream, stage, area):
+def estimateTiledAcquisitionTime(stream, area):
```

Comment:
Note that the reason I had put stage is that I like having the "time estimation" function take *exactly* the same arguments as the actual function. This way, it's simple to know that it will estimate the right acquisition. In particular here, we don't use the stage now, but *could* use it if we cared about the speed of move of the axes.

Reply:
Makes sense. I put it back. What is the "actual function" you are referring to? The `estimateTiledAcquisitionTime` in `_tiledacq.py` takes just `*args, **kwargs`.